### DISCUSSION TITLE
Many to Many field: automatic sorting by field

### DISCUSSION THREAD

**Discussion by dreerr** [directus/directus#10399] [Feature Requests]
I am looking for a functionality that will automatically sort the relational data with an existing field in the interface. Now one can designate a sorting field for custom ordering of items, but I am looking for a possibility to sort for example all related people of a company by last name in the admin interface.

<img width="823" alt="Bildschirmfoto 2021-12-08 um 19 22 14" src="https://user-images.githubusercontent.com/1454661/145262693-ccaec3ad-f047-42c2-8776-881b146ddd63.png">

**Comment by rijkvanzanten** [comment]
100%. Wondering if we can add that in as part of the https://github.com/directus/directus/pull/9135 sprint

**Reply by dreerr** [reply]
That would be incredibly great 🤩

**Reply by dreerr** [reply]
Dear @rijkvanzanten 
I was just wondering if it could make it in the next sprint 😇

**Reply by rijkvanzanten** [reply]
Hopefully, though I can't guarantee anything.. Would you like to add a PR to implement it?

---

_Obligatory message about this._

[**Unfortunately, we can not provide estimates on features or fixes**](https://docs.directus.io/getting-started/support.html#can-you-give-an-eta-for-this-feature-fix). As of now, the only way to get an accurate timeline or release date is to [sponsor](https://docs.directus.io/getting-started/support.html#sponsored-work) this task. We recognize that this may be an important feature/fix, but we are a small open-source organization with a lot to triage and complete. If you can't sponsor this ticket, then the next best thing is to increase its priority by giving it a 👍 . Or contribute a pull-request... after all, this is OSS!

**Comment by dreerr** [comment]
Dear @rijkvanzanten maybe I could do a PR, could you point me into the right direction here?

**Reply by rijkvanzanten** [reply]
The interface code can be found here: https://github.com/directus/directus/tree/main/app/src/interfaces/list-m2m

The values are fetched here:

https://github.com/directus/directus/blob/66b8e1f3d3721487385e26aa9728221303f39e3e/app/src/interfaces/list-m2m/list-m2m.vue#L197-L205

The parameters for the preview values request are set here: https://github.com/directus/directus/blob/66b8e1f3d3721487385e26aa9728221303f39e3e/app/src/interfaces/list-m2m/use-preview.ts#L206-L211

**Comment by giovanni-ciccia** [comment]
Hi, do you have any update on this functionality?

**Reply by rijkvanzanten** [reply]
It's an open feature request 👍🏻

**Reply by ricricucit** [reply]
I was hunting for "manual sorting dragn&drop" and cannot find if that exists or not, BUT

 ✅ **I can confirm that this feature now exists Directus (9.X).**

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
